### PR TITLE
Navigation by select and click

### PR DESCRIPTION
**Pull request by netsquire**
Please, review!

Navigation part of tag editing.

### CODE REVIEW

**Comment by netsquire** [comment]
Added 'set_tag'

**Comment by netsquire** [comment]
Added 'Remove all tags'

File: navigation/navigation.py [hunk 1]
```diff
@@ -210,3 +216,110 @@ def found_by_pattern(self, pattern):
...
+        click1 = driver.find_element_by_xpath("//button[@data-toggle='dropdown'][@data-id='tag_cat']")
+        self.power_click(click1)
+
+        all_options = driver.find_elements_by_xpath("//select[@id='tag_cat']/../div[contains(@class, 'btn-group')]/div/ul/li/a")
+        for option in all_options:
+            if tag_name in option.text:
+                print "Tag_name is substring of : ", option.text
+                sleep(1)
+                self.go_up_till_clickable(option)
+        sleep(3)
```
Comment (mattmahoneyrh) [inline navigation/navigation.py]: Is there a more dynamic check that can be done here, instead of using a sleep?

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [177](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/177/)
<br> Gif records:<br>

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [178](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/178/)
<br> Gif records:<br>

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [183](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/183/)
<br> Gif records:<br>

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [202](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/202/)
<br> Gif records:<br> - [record_test_datasource.flv](http://10.16.23.184/records/202/record_test_datasource.flv) <br> - [record_test_deployments.flv](http://10.16.23.184/records/202/record_test_deployments.flv) <br> - [record_test_domains.flv](http://10.16.23.184/records/202/record_test_domains.flv) <br> - [record_test_providers.flv](http://10.16.23.184/records/202/record_test_providers.flv) <br> - [record_test_servers.flv](http://10.16.23.184/records/202/record_test_servers.flv) <br> - [record_test_topology.flv](http://10.16.23.184/records/202/record_test_topology.flv) <br>

**Comment by netsquire** [comment]
would complete this task in another branch/pr

**Pull request closed without merging**